wifiutils/src/main/java/com/thanosfisherman/wifiutils/ConnectorUtils.java

```diff
@@ -488,6 +468,35 @@ public void onFailed(int reason) {
         wifiManager.startWps(wpsInfo, wpsCallback);
     }
 
+    @RequiresPermission(ACCESS_WIFI_STATE)
+    static boolean disconnectFromWifi(@NonNull final Context context, @NonNull final ConnectivityManager connectivityManager, @NonNull final WifiManager wifiManager) {
+        if (isAndroidQOrLater()) {
+            return disconnectAndroidQ(connectivityManager);
+        }
+
+        return wifiManager.disconnect();
+    }
+
+    static boolean removeWifi(@NonNull final Context context, @NonNull final ConnectivityManager connectivityManager, @NonNull final WifiManager wifiManager, @NonNull final String ssid) {
+        if (isAndroidQOrLater()) {
+            return disconnectAndroidQ(connectivityManager);
+        }
+
+        final WifiConfiguration wifiConfiguration = ConfigSecurities.getWifiConfiguration(wifiManager, ssid);
+        return cleanPreviousConfiguration(wifiManager, wifiConfiguration);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.Q)
+    static boolean disconnectAndroidQ(@NonNull final ConnectivityManager connectivityManager) {
+        if (networkCallback != null)
+        {
```

Comment:
Additionally please put this curly brace at the end of the if so that the format is aligned with the rest of the code.